packages/framework/tests/Unit/Component/Domain/DomainTest.php

```diff
@@ -10,28 +10,47 @@
 
 class DomainTest extends TestCase
 {
-    public function testGetIdNotSet()
+    /**
+     * @return \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig
+     */
+    private function createDomainConfigFirst(): DomainConfig
     {
-        $domainConfigs = [
-            new DomainConfig(1, 'http://example.com:8080', 'example', 'cs'),
-            new DomainConfig(2, 'http://example.org:8080', 'example.org', 'en'),
+        return new DomainConfig(Domain::FIRST_DOMAIN_ID, 'http://example.com:8080', 'example.com', 'cs');
```

Comment:
Is OK that you renamed example to example.com?

Reply:
The original test expects exception to be thrown so it is OK.

Reply:
The original test expects exception to be thrown so it is OK.